btree/btree.c

```diff
@@ -3529,21 +3566,417 @@ int64_t btree_iter_tick(struct m0_sm_op *smop)
 	}
 }
 
-int64_t btree_del_tick(struct m0_sm_op *smop)
+/* Delete Operation */
+
+/**
+ * This function will get called if there is an underflow at current node after
+ * deletion of the record. Currently, underflow condition is defined based on
+ * record count. If record count is 0, there will be underflow. To resolve
+ * underflow,
+ * 1) delete the node from parent.
+ * 2) check if there is an underflow at parent due to deletion of it's child.
+ * 3) if there is an underflow,
+ *        if, we have reached root, handle underflow at root.
+ *        else, repeat steps from step 1.
+ *    else, return next phase which needs to be executed.
+ *
+ * @param bop it will provide all required information about btree operation.
+ * @return int64_t it return state which needs to get executed next.
+ */
+static int64_t m0_btree_del_resolve_underflow(struct m0_btree_op *bop)
```

Comment:
Coding Style. 
Functions which are not static, not globally exported, and are shared only across multiple files within a module - shall be prefixed with m0_<module-name>__

Reply:
okay.